Java/queue/queue.java

```diff
@@ -0,0 +1,79 @@
+// Java program for linked-list implementation of queue
+
+// A linked list (LL) node to store a queue entry
+class QNode {
```

Comment:
File name and class should be same in a Java file, kindly make the required changes immediately

Reply:
@Kumar-laxmi In my program there are two class one is  QNode and other is Queue .So file name is according to second one. Is not it ok?
Please explain if not.

Reply:
ok